src/decanter/core/core_api/api.py

```diff
@@ -118,6 +118,18 @@ def get_data_file_by_id(self, data_id):
         """
         return self.requests_(http='GET', url='/data/%s/file' % data_id)
 
+    def get_ts_algorithms(self):
```

Comment:
The CoreAPI class is a wrapper of the Core service. Every function in this class stands for each endpoint. In this class, we should make the function name like method_endpoint or method_endpoint_by_sth. Also, keep the original resp from the Core service. Try not to update/modify the result before returning.

Reply:
What is returned from the get request is a dictionary of dictionaries that is hard for the user to interpret. In this situation, it is reasonable to re-format the data returned from the get request in a manner that is easy for the client to understand and process.

I can change the name of the method to get_info(self)